packages/components/src/popover/stories/index.tsx

```diff
@@ -128,19 +128,6 @@ Default.args = {
 	),
 };
 
-export const Toolbar: ComponentStory< typeof Popover > = Template.bind( {} );
```

Comment:
This Storybook example is being removed since we don't want to incentivize the usage of the `variant: "toolbar"` prop on the `Popover` component. The prop is still around and can be tweaked in all Storybook examples — there simply won't be an example dedicated to the toolbar variant.